### PR TITLE
add canary vs nightly explanation command

### PR DESCRIPTION
**Pull request by BreadFish64**

### CODE REVIEW

File: src/responses.json [hunk 1]
```diff
@@ -18,5 +18,6 @@
     "forum": { "reply": "This question might be more suitable for the *Citra* forum. <https://community.citra-emu.org/>"},
     "game": { "reply": "Click here to view our game compatibility list: <https://citra-emu.org/game/>"},
     "log": { "reply": "This forum topic tells you how to __get the log file__: <https://community.citra-emu.org/t/how-to-upload-the-log-file/296>"}
```
Comment (BynariStar) [inline src/responses.json]: Need to add a comma after the `log` object

File: src/responses.json [hunk 2]
```diff
@@ -18,5 +18,6 @@
...
+    "canary": { "reply": "The nightly build of Citra contains already reviewed and tested features. If you require support with the installation or use of Citra, or you want to report bugs you should use this version.\nThe Canary build of Citra is the same as our nightly builds, with additional features that are still waiting on review before making it into the official Citra builds. We will not provide support for issues found only in this version. If you believe you’ve found a bug, please retest on our nightly builds. Both versions are still in development, so expect crashes and bugs."},
```
Comment (BynariStar) [inline src/responses.json]: Need to remove the comma after the `canary` object

File: src/responses.json [hunk 3]
```diff
@@ -17,6 +17,7 @@
...
     "forum": { "reply": "This question might be more suitable for the *Citra* forum. <https://community.citra-emu.org/>"},
     "game": { "reply": "Click here to view our game compatibility list: <https://citra-emu.org/game/>"},
-    "log": { "reply": "This forum topic tells you how to __get the log file__: <https://community.citra-emu.org/t/how-to-upload-the-log-file/296>"}
+    "log": { "reply": "This forum topic tells you how to __get the log file__: <https://community.citra-emu.org/t/how-to-upload-the-log-file/296>"},
+    "canary": { "reply": "The nightly build of Citra contains already reviewed and tested features. If you require support with the installation or use of Citra, or you want to report bugs you should use this version.\nThe Canary build of Citra is the same as our nightly builds, with additional features that are still waiting on review before making it into the official Citra builds. We will not provide support for issues found only in this version. If you believe you’ve found a bug, please retest on our nightly builds. Both versions are still in development, so expect crashes and bugs."}
```
Comment (Dragios) [inline src/responses.json]: ...installation or use...
Replace to usage

We will not provide...
nit: Replace `will` with `do`

Usage of `only` is kind of redundant in this context.

Comment (BreadFish64) [reply src/responses.json]: I disagree with the last point, users will find a way to misconstrue it if we leave something out, this text is also nearly a direct copy of the download page, so if you want it changed here...

**Pull request merged**